### ISSUE TITLE
Vagrant issues

### ISSUE THREAD

**Issue by jseaidou** [oneops/setup#23]
following the guidelines: I ran into a few issues that I tried to resolve but can't get the login page to work.
running `sudo service display start gives:`
```/etc/init.d/display: line 21: /opt/oneops/log/display.log: No such file or directory```

after creating the log directory, `sudo service display start` and succeeding,
```tail -f /opt/oneops/log/rails.log``` outputs
```
Could not find proper version of railties (4.1.10) in any of the sources
Run `bundle install` to install missing gems.
```

The output of ```bundle install``` gives:
```
An error occurred while installing clipboard-rails (1.5.8), and Bundler cannot continue.
Make sure that `gem install clipboard-rails -v '1.5.8'` succeeds before bundling.
```

Following the output I ran: ``` gem install clipboard-rails -v '1.5.8'``` and got the output:
```
ERROR:  Error installing clipboard-rails:
        clipboard-rails requires Ruby version >= 2.0.0.
```
And this just leads down a bigger rabbit hole where inductor stop working

**Comment by kfilipovski** [comment]
@jseaidou i merged https://github.com/oneops/setup/pull/22 earlier today which has the fix for this issue, but it requires a new vagrant instance since the fix is upgrading rvm ruby to 2.0.0.

**Comment by jseaidou** [comment]
Thanks @kfilipovski that seems to have resolved it
